pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -104,6 +105,10 @@ public String getInstance(
     response.set("tags", JsonUtils.objectToJsonNode(instanceConfig.getTags()));
     response.set("pools", JsonUtils.objectToJsonNode(instanceConfig.getRecord().getMapField(InstanceUtils.POOL_KEY)));
     response.put("grpcPort", getGrpcPort(instanceConfig));
+    String queriesDisabled = instanceConfig.getRecord().getSimpleField(CommonConstants.Helix.QUERIES_DISABLED);
+    if ("true".equals(queriesDisabled)) {
```

Comment:
I think that it's better to handle upper/lower cases:
```
e.g.

queriesDisabled = true
queriesDisabled = True <- By default, Python uses this pattern for boolean and we had an issue with this once. 
queriesDisabled = TRUE
...
```

Reply:
I'll update it with `equalsIgnoreCase()` to make it more robust.